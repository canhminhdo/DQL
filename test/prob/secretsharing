#!/bin/sh

DIR=../../test/prob
cd ../../examples/prob && maude < secretsharing.maude -no-banner -no-advise | sed -r '/^(rewrites:|Solution|states:)/d' > $DIR/secretsharing.out
diff $DIR/secretsharing.expected $DIR/secretsharing.out
rm $DIR/secretsharing.out