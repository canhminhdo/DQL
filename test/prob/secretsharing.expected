==========================================
reduce in SECRET-SHARING : initQState |= [H(1) ; CX(1,2) ; CX(0,1) ; H(0) ; CX(
    2,3) ; H(2) ; ((P(1,|0>) ? with prob == 1/2) ; skip U (P(1,|1>) ? with prob
    == 1/2) ; X(3)) ; ((P(0,|0>) ? with prob == 1/2) ; skip U (P(0,|1>) ? with
    prob == 1/2) ; Z(3)) ; ((P(2,|0>) ? with prob == 1/2) ; skip U (P(2,|1>) ?
    with prob == 1/2) ; Z(3))]P(3,qubitAt(initQState, 0)) .
result JudgmentSet: emptyJS
Bye.
